lib/gris/grape_extensions/date_time_helpers.rb

```diff
@@ -0,0 +1,34 @@
+require 'chronic'
+
+module Gris
+  module DateTimeHelpers
+    extend ActiveSupport::Concern
+
+    # define keys that need to be converted from strings
+    # to datetime via Chronic
+    def datetime_params(*keys_to_convert)
+      @datetime_keys_to_convert = Set.new(keys_to_convert)
+    end
+
+    def keys_to_convert
+      @datetime_keys_to_convert
+    end
+
+    def process_datetime_params(params)
+      params.keys.each do |key|
+        if keys_to_convert.include?(key)
+          params[key] = string_to_datetime(params[key])
+        end
+      end
+    end
+
+    def string_to_datetime(string)
+      return string if string.blank?
```

Comment:
We could skip this and the re-setting of `params[key]` with nil/'' values altogether if we know that `params[key].blank?` in `process_datetime_params(params)`

in https://github.com/dylanfareed/gris/pull/26/files#diff-202747fde68a6c2364d14c8c71fd1083R20

Minor and maybe inconsequential, tho!